### PR TITLE
Pointer check beautification

### PR DESCRIPTION
**Pull request by kroening**
- [X] Each commit message has a non-empty body, explaining why the change was made.
- [X] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by thk123 — APPROVED** [review]

File: src/analyses/goto_check.cpp [hunk 1]
```diff
@@ -170,7 +170,18 @@ class goto_checkt
   void undefined_shift_check(const shift_exprt &, const guardt &);
   void pointer_rel_check(const exprt &, const guardt &);
   void pointer_overflow_check(const exprt &, const guardt &);
-  void pointer_validity_check(const dereference_exprt &, const guardt &);
+
+  /// Triggers the generation of VCCs for the validtity
```
Comment (smowton) [inline src/analyses/goto_check.cpp]: Sp validtity

Comment (smowton) [inline src/analyses/goto_check.cpp]: Triggers the generation -> Generates?

**Review by smowton — COMMENTED** [review]
Please add a test

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: dc734f761f3502b995c0326ce529dcd70db5a65c).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/117279453

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: ab1c30b0dfb6dec65ad11d8b911e04f258b13b3e).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/117287587

**Pull request merged**